pkg/envoy/envoy.go

```diff
@@ -39,10 +39,9 @@ import (
 var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "envoy-manager")
 
 var (
-	// Running Envoy version will be checked against `RequiredEnvoyVersionSHA`.
-	// By default cilium-agent will fail to start if there is a version mismatch.
-	// This value must match the one in file CILIUM_ENVOY_SHA.
-	RequiredEnvoyVersionSHA = "fd3866831d8f85e27120f34d98133343fc9c4e0e"
+	// RequiredEnvoyVersionSHA is set during build
+	RequiredEnvoyVersionSHA string
```

Comment:
Can you keep the log message about how `cilium-agent` will fail to start if there is a version mismatch?